### PR TITLE
ENH: Add spaghetti to meta-package

### PR DESCRIPTION
**Pull request by sjsrey**

### CODE REVIEW

**Comment by jGaboardi** [comment]
@sjsrey will #1047 work without spaghetti being up on pypi?

**Comment by sjsrey** [comment]
@jGaboardi yes it will as this is a source distribution that is pulling from master on github not pypi.

File: .travis.yml [hunk 1]
```diff
@@ -34,6 +34,9 @@ script:
   - python setup.py sdist >/dev/null
   - echo "check_stable=False" >pysal/config.py
   - travis_wait 45 nosetests --verbose --with-coverage --cover-package=pysal;
+  - nosestests --verbose --with-coverage pysal/dynamics
+  - nosestests --verbose --with-coverage pysal/explore
```
Comment (ljwolf) [inline .travis.yml]: I would like to check if `pytest` does this right.

Comment (sjsrey) [reply .travis.yml]: Worth a shot. If pytest isn't the solution, we can always go with the explicit approach here.

**Pull request closed without merging**